logger.py

```diff
@@ -0,0 +1,70 @@
+"""A module to write logs and files."""
```

Comment:
Can you add some comments here to explain what this module is for, and why it's required?